doc/6/getting-started/vuejs/with-vuex/src/store.js

```diff
@@ -0,0 +1,68 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+  state: {
+    messages: [],
+    roomID: '',
+    nickname: ''
+  },
+  actions: {
+    INIT: async ({ dispatch }, { kuzzle }) => {
+      await kuzzle.connect();
+      if (!(await kuzzle.index.exists("chat"))) {
+        await kuzzle.index.create("chat");
+        await kuzzle.collection.create("chat", "messages");
+      }
+      dispatch('FETCH_MESSAGES', {kuzzle});
+    },
+    SEND_MESSAGE: async ({ state }, { kuzzle, message }) => {
+      await kuzzle.document.create("chat", "messages", {
+        value: message,
+        username: state.nickname
+      });
+    },
+    FETCH_MESSAGES: async ({ commit, dispatch }, { kuzzle }) => {
+      const results = await kuzzle.document.search(
+        "chat",
+        "messages",
+        { sort: ["_kuzzle_info.createdAt"] },
+        { size: 100 }
+      );
+      results.hits.map(hit => {
+        commit('ADD_MESSAGE', hit);
+      });
+      dispatch('SUBSCRIBE_MESSAGES', {kuzzle});
+    },
+    SUBSCRIBE_MESSAGES: async ({ commit }, { kuzzle }) => {
+      const roomID = await kuzzle.realtime.subscribe("chat", "messages", {}, notification => {
+        if (
+          notification.type === "document" &&
+          notification.action === "create"
+        ) {
+        commit('ADD_MESSAGE', notification.result);
+        }
+      });
+      commit('SET_ROOMID', roomID);
```

Comment:
You set the roomId in the store but you never use it. I think you should unsubscribe this roomId at some point (when the route change for example)

Reply:
Effectively, I did this in the other vuejs [gettings-started](https://github.com/kuzzleio/sdk-javascript/blob/b8b2c35055a6fde2593e394572b47178acc2f7c0/doc/6/getting-started/vuejs/without-vuex/src/App.vue), and i explained why I set the roomId in comments because in this little app I don't think its better to weighing the router. 
Should I change that in both vuejs getting-started ?

Reply:
I'm just worried people would not understand why we do this in the getting started so I think it would be better, yes.

Reply:
Okay I'll do the changes :)